Makefile

```diff
@@ -2,7 +2,7 @@
 # Project build information
 #------------------------------------------------------------------
 PROJNAME := kubernetes-toolkit
-KUBERNETES_VERSION := 1.18.14
+KUBERNETES_VERSION := 1.21.00
```

Comment:
I would either go with the latest `1.20` release which is `1.20.14`  or the most recent release altogether which is `1.23.3`

Reply:
I went wit 1.21.8